datalad/support/gitrepo.py

```diff
@@ -1680,7 +1736,10 @@ def get_merge_base(self, commitishes):
         if not commitishes:
             raise ValueError("Provide at least a single value")
         elif len(commitishes) == 1:
-            commitishes = commitishes + [self.get_active_branch()]
+            branch = self.get_active_branch()
+            if branch is None:
+                raise ValueError("Single commitish provided and no active branch")
```

Comment:
LGTM, Note: in prior cases we would have got `TypeError` though but since it is indeed illegit and unlikely invocation , should be ok. Just FTR

```
❯ python -c 'from datalad.support.gitrepo import GitRepo; print(GitRepo(".").call_git_oneline(["merge-base", "master", None]))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/yoh/proj/datalad/datalad-maint/datalad/dataset/gitrepo.py", line 550, in call_git_oneline
    lines = list(self.call_git_items_(args, files=files,
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yoh/proj/datalad/datalad-maint/datalad/dataset/gitrepo.py", line 517, in call_git_items_
    for file_no, line in self._generator_call_git(
  File "/home/yoh/proj/datalad/datalad-maint/datalad/dataset/gitrepo.py", line 345, in _generator_call_git
    generator = self._git_runner.run(
                ^^^^^^^^^^^^^^^^^^^^^
  File "/home/yoh/proj/datalad/datalad-maint/datalad/runner/runner.py", line 206, in run
    results_or_iterator = threaded_runner.run()
                          ^^^^^^^^^^^^^^^^^^^^^
  File "/home/yoh/proj/datalad/datalad-maint/datalad/runner/nonasyncrunner.py", line 343, in run
    return self._locked_run()
           ^^^^^^^^^^^^^^^^^^
  File "/home/yoh/proj/datalad/datalad-maint/datalad/runner/nonasyncrunner.py", line 403, in _locked_run
    self.process = Popen(self.cmd, **kwargs)         # nosec
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1834, in _execute_child
    self.pid = _fork_exec(
               ^^^^^^^^^^^
TypeError: expected str, bytes or os.PathLike object, not NoneType
```